### PR TITLE
planning to make sth like this:

### PR DESCRIPTION
**Pull request by mdayican**
### Change description ###

No need to review now!!

use existing processor as a process template and for new message , Paymentoperation would define handlers

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/servicebus/PaymentOperation.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    public static PaymentOperation valueFromStr(String operation) {
+        try {
+            return PaymentOperation.valueOf(operation);
+        } catch (IllegalArgumentException ex) {
+            throw new IllegalPaymentOperationException("Operation:" + operation + " is not a valid Payment operation");
+        }
+    }
+
+}
```
Comment (rkondratowicz) [inline src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/servicebus/PaymentOperation.java]: we can avoid this entire class by using a simple switch statement in the `PaymentMessageProcessor`

File: src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/servicebus/handler/PaymentMessageHandler.java [hunk 2]
```diff
@@ -32,14 +33,15 @@ public PaymentMessageHandler(
         this.payHubClient = payHubClient;
     }
 
-    public CreatePaymentResponse handlePaymentMessage(CreatePaymentMessage paymentMessage) {
-        CreatePaymentRequest request = paymentRequestMapper.mapPaymentMessage(paymentMessage);
+    public void handlePaymentMessage(PaymentMessage paymentMessage) {
+        CreatePaymentMessage createPaymentMessage = (CreatePaymentMessage)paymentMessage;
```
Comment (rkondratowicz) [inline src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/servicebus/handler/PaymentMessageHandler.java]: just take `CreatePaymentMessage` as argument

File: src/test/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/PaymentMessageProcessorTest.java [hunk 3]
```diff
@@ -105,6 +105,15 @@ public void should_not_throw_exception_when_queue_message_is_invalid() throws Ex
         assertThat(paymentMessageProcessor.processNextMessage()).isTrue();
     }
 
+    @Test
+    public void should_not_throw_exception_when_queue_message_label_is_invalid() throws Exception {
+        IMessage invalidMessage = mock(IMessage.class);
+        given(invalidMessage.getLabel()).willReturn("CREATE","CREATE","SET");
+        given(messageReceiver.receive()).willReturn(invalidMessage);
+
+        assertThat(paymentMessageProcessor.processNextMessage()).isTrue();
```
Comment (rkondratowicz) [inline src/test/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/PaymentMessageProcessorTest.java]: this only takes the firlst message and `"CREATE"` is valid

**Comment by mdayican** [comment]
continued with #82

**Pull request closed without merging**